content-api-post.js

```diff
@@ -27,7 +27,7 @@ exports.handler = function (event, context, callback) {
   const data = JSON.parse(event.body);
   const decodedImage = Buffer.from(data.body, 'base64');
 
-  if (!decodedImage || !data.accountId || !data.type) {
+  if (!decodedImage || (!data.accountId && !data.championFundId) || !data.type) {
```

Comment:
@NoelLH This should now fail if both are empty, and will pass if either 1 is true